.github/workflows/build.yaml

```diff
@@ -0,0 +1,25 @@
+name: build
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    env:
+      TERRAFORM_VERSIONS: "1.0.5 0.14.5 0.13.6 0.12.30 0.11.14"
+    strategy:
+      fail-fast: true
+      matrix:
+        ruby: [ '2.4', '2.5', '2.6', '2.7', '3.0' ]
```

Comment:
Ruby 2.4 had EOL over 1.5 years ago, and 2.5 end of March.

I think we could save some cycles and drop testing of at least for the v2.4. What do you think?
But this can be done as a separate PR, as it also affect README and rubocop configurations, and might lead to some syntax changes.